v2/README.md

```diff
@@ -3,6 +3,9 @@ title: Azure Service Operator v2
 type: docs
 description: "Manage your Azure resources from within your Kubernetes cluster."
 ---
+
+<img src="https://azure.github.io/azure-service-operator/favicons/favicon-256.png" style="float:right; "/>Azure Service Operator (ASO) allows you to deploy and maintain Azure Resources using the Kubernetes tooling you already know and use. Instead of separately deploying your [Redis Cache](https://azure.github.io/azure-service-operator/reference/cache/) or [PostgreSQL database server](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/) and then configuring your Kubernetes application to use them, you can use ASO to deploy and manage these resources from within your Kubernetes cluster. ASO will then automatically configure your application to use them.
```

Comment:
> Instead of separately deploying your [Redis Cache](https://azure.github.io/azure-service-operator/reference/cache/) or [PostgreSQL database server](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/) and then configuring your Kubernetes application to use them

Why only mentioning about the two resources on the main page?  IMO this can confuse the user at first, that we mainly support Redis and Postgresql

Reply:
Agree w/ Harsh - we should make it clearer that these are examples (or possibly just not give examples?) Possibly:
```suggestion
<img src="https://azure.github.io/azure-service-operator/favicons/favicon-256.png" style="float:right; "/>Azure Service Operator (ASO) allows you to deploy and maintain Azure Resources using the Kubernetes tooling you already know and use. Instead of deploying and managing your Azure resources separate from your Kubernetes application you can manage them together. ASO can automatically configure your application to integrate with the deployed Azure resources. For example you could manage your [Redis Cache](https://azure.github.io/azure-service-operator/reference/cache/) or [PostgreSQL database server](https://azure.github.io/azure-service-operator/reference/dbforpostgresql/) and have ASO help configure your Kubernetes application to use them.
```